pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -74,11 +74,13 @@ class GAMSDirect(pyomo.common.plugin.Plugin):
     Visit Python API page on gams.com for installation help.
     """
     pyomo.common.plugin.implements(IOptSolver)
-    pyomo.common.plugin.alias('_gams_direct', doc='The GAMS modeling language')
+    pyomo.common.plugin.alias('_gams_direct',
+        doc='Direct python interface to the GAMS modeling language')
 
     def __init__(self, **kwds):
         self._version = None
         self._default_variable_value = None
+        self._metasolver = False
```

Comment:
I thought the GAMS interface is a metasolver?

Reply:
To be honest I had no idea so I took a guess. What's a metasolver?

Reply:
A solver that functions mostly to call other solvers. Included are the solver interfaces, like `asl` and `py`.

Actually, looking at the list above, I think GDPopt qualifies as a metasolver as well.

Reply:
Currently the only `_metasolver=True` solvers are bilevel, mpec, and asl.

Reply:
Actually the help message below the solver list says `A plus indicates meta-solvers, that are always available`, but gams is definitely not always available. In the code for this help function, it decides to show a `+` if it's a metasolver before it decides to show a `*` if it's `available`. Currently also your GDPopt returns True for available no matter what.